### ISSUE TITLE
Fails to run on Windows due to terminal pretty printing

### ISSUE THREAD

**Issue by jeffgerhard** [unt-libraries/pycallnumber#21]
Hi,

This doesn't currently work in Windows because the utils script requires POSIX-only features (fcntl and termios). I did a quick edit in a fork to nicely resolve this for my use case, but I'm sure there is a more elegant solution. Happy to do a pull request if you like my fast solution, though. Thanks for the useful script!

**Comment by pybrarian** [comment]
I just issued a pull on this same issue before seeing this issue (I started the pull before the issue was up).

**Comment by jthomale** [comment]
Okay. Apologies all around for leaving this hanging for a week; things have been a little crazy...

I don't normally develop in anything but a *nix environment, but mea culpa for not doing any testing at all on Windows before releasing this and presenting at Code4lib. It was a complete oversight. Thanks to both of you (@jeffgerhard and @pybrarian) for raising the issue and offering fixes.

@pybrarian: I've spent part of the afternoon testing your PR on multiple Python versions both on Windows and Unix, and I think your solution works and looks good. 👍 Tests are passing for me as well and the pretty-printing functionality works as expected in all the environments I've tested. There are a couple of (very!) minor stylistic points I'll ask you to address before I'm ready to merge, solely for the sake of consistency within this project, but I'll add those comments onto the PR itself. Thanks so much!
